src/libp2pConfigs/getDhtLibp2pConfig.ts

```diff
@@ -8,43 +8,10 @@ export const getDhtLibp2pConfig = (): Libp2pOptions => ({
   dht: kadDHT({
     clientMode: true
   }),
-  /**
-     * @see https://github.com/libp2p/js-libp2p/blob/master/doc/CONFIGURATION.md#configuring-connection-manager
-     */
-  connectionManager: {
-    maxConnections: Infinity,
-    minConnections: 1,
```

Comment:
You want `minConnections` higher than this - as set it'll only ensure you have a connection to 1x node, it should be greater than the number of bootstrappers at least!